methods/eps_fhir/api_methods.py

```diff
@@ -0,0 +1,125 @@
+import json
+import uuid
+
+from features.environment import CIS2_USERS
+from methods.eps_fhir.api_request_body_generators import (
+    create_fhir_bundle,
+    create_fhir_parameter,
+    generate_message_header,
+    generate_medication_request,
+    generate_patient,
+    generate_organization,
+    generate_practitioner_role,
+    generate_practitioner,
+    generate_provenance,
+    generate_agent,
+    generate_owner,
+    generate_group_identifier,
+)
+from methods.shared.common import the_expected_response_code_is_returned
+from methods.shared.api import post, get_default_headers
+from utils.prescription_id_generator import generate_short_form_id
+from utils.signing import get_signature
+
+
+def create_new_prepare_body(context):
```

Comment:
Generally, functions that are internal to a file are prefixed with an underscore, to make their usage clear.